### PR TITLE
Automation scripts for the SCSS package

### PR DESCRIPTION
**Pull request by sanlucca**
Added gulp automation for SCSS.
Added css linting to ensure syntax coherence throughout the project.

Refactored all scss files to be adequate to linting rules.

### CODE REVIEW

File: packages/scss/gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	gulp.watch("./demo/*.html", browserSync.reload);
+});
+
+gulp.task('sass:debug', () => {
+	return gulp.src('./src/main.scss')
+	.pipe(sass(SASS_OPTIONS_DEBUG).on('error', sass.logError))
+	.pipe(rename('lucca-front.css'))
+	.pipe(autoprefixer(AUTOPREFIXER_OPTIONS))
+	.pipe(gulp.dest(OUT_DIR))
+	.pipe(browserSync.stream());
```
Comment (GuillaumeNury) [inline packages/scss/gulpfile.js]: 👍

File: packages/scss/package.json [hunk 2]
```diff
@@ -29,7 +29,14 @@
...
   "devDependencies": {
+    "browser-sync": "^2.18.8",
+    "gulp": "^3.9.1",
+    "gulp-autoprefixer": "^3.1.1",
+    "gulp-clean": "^0.3.2",
+    "gulp-rename": "^1.2.2",
+    "gulp-sass": "^3.1.0",
+    "gulp-stylelint": "^3.9.0",
     "node-sass": "^4.5.2",
     "nodemon": "^1.11.0"
```
Comment (GuillaumeNury) [inline packages/scss/package.json]: `nodemon` is not usefull anymore. Does `node-sass` is still needed as it is a dependancy of `gulp-sass` ?

Comment (sanlucca) [inline packages/scss/package.json]: Yes, you're right. I stripped out both.

**Comment by lucienbertin** [comment]
can i merge ?

**Pull request merged**